src/components/consul/detail/ConsulQuestionsListItem.js

```diff
@@ -47,22 +60,33 @@ export const ConsulQuestionsListItem = ({ questionItem, refetch, token, disabled
           key={index}
           style={[
             styles.answerContainer,
-            answersGivenByCurrentUser[0] &&
-              answersGivenByCurrentUser[0].answer === item.title &&
-              styles.selectedContainer,
-            !disabled &&
+            (resultsReadyToBeShown && highestVote === item.totalVotesPercentage) ||
+            (!resultsReadyToBeShown &&
+              answersGivenByCurrentUser[0] &&
+              answersGivenByCurrentUser[0].answer === item.title)
```

Comment:
`answersGivenByCurrentUser[0] && answersGivenByCurrentUser[0].answer === item.title` could be changed to `answersGivenByCurrentUser?.[0]?.answer === item.title` to be sure, that the first entry exists with an answer to compare.

(same for lines 71+72)

Reply:
Yes, I wanted to fix that. But I guess we don't have enough time for this Ticket anymore.

Reply:
i understand. but this refactoring can be done before merge.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/541/commits/a6c542a703a69d728f401ebc936cfebf6ac23300